src/openrct2/ride/Ride.cpp

```diff
@@ -904,8 +936,12 @@ bool Ride::SupportsStatus(RideStatus s) const
  */
 void RideInitAll()
 {
-    _rides.clear();
-    _rides.shrink_to_fit();
+    std::for_each(std::begin(_rides), std::end(_rides), [](auto& ride) {
+        ride.id = RideId::GetNull();
+        ride.type = RIDE_TYPE_NULL;
+        ride.custom_name = {};
```

Comment:
should we also do measurement = {}? The fact this is the same as RideDelete makes me think it should be extracted out to a method.

Reply:
Agree and done.